Integrations/Pwned-V2/Pwned-V2.py

```diff
@@ -181,12 +189,25 @@ def domain_to_entry_context(domain, hibp_res):
         dbot_score = DEFAULT_DBOT_SCORE_DOMAIN
         domain_context = create_context_entry('domain', domain, comp_sites, [], DEFAULT_DBOT_SCORE_DOMAIN)
         comp_domain[outputPaths['domain']] = domain_context
+        comp_domain['Account.Domain'] = domain
 
     comp_domain['DBotScore'] = create_dbot_score_dictionary(domain, 'domain', dbot_score)
 
     return comp_domain
 
 
+def rate_limit_retry(amount_of_seconds, request_type):
+    time.sleep(amount_of_seconds)
+    if request_type == 'email':
+        pwned_email_command()
+    else:
+        pwned_domain_command()
+
+
+def retry_needed(api_res, api_paste_res):
+    return api_res and api_paste_res and ('request_retry' in api_res or 'request_retry' in api_paste_res)
```

Comment:
split into two conditions, this usage of api_paste_res which doesn't always exists is weird